src/api.c

```diff
@@ -5267,6 +5283,18 @@ int cgroup_get_controller_next(void **handle, struct cgroup_mount_point *info)
 		goto out_unlock;
 	}
 
+	if (strncmp(cg_mount_table[*pos].name, CGROUP_FILE_PREFIX, CONTROL_NAMELEN_MAX) == 0)
```

Comment:
This change was needed to appease `lscgroup`.  The `cgroup` controller was showing up in its lists of controllers, and I don't think this is desired behavior